src/utils/XhrNetworkInterceptor.ts

```diff
@@ -53,8 +60,85 @@ const _reset = () => {
     gqlQueryName: '',
     serverErrorMessage: '',
     requestContentType: '',
+    isW3cHeaderFound: null,
+    partialId: null,
+    networkStartTimeInSeconds: null,
+    w3cGeneratedHeader: null,
+    w3cCaughtHeader: null,
   };
 };
+const getFeatureFlags = async (networkData: NetworkData) => {
```

Comment:
Why is this function called `getFeatureFlags` when what it actually does is generate the `traceparent` header? I feel like we could rename it to something more meaningful, what do you think?